tools/travis/docker_run.sh

```diff
@@ -11,6 +11,7 @@ eval "$(pyenv init -)"
 pyenv global ${PYVER}
 pyenv local ${PYVER}
 
+python -m pip install tensorflow
```

Comment:
Because of this, can we check that builds are not slowed down a lot ? (just looking at the timing of the builds on travis). If it takes a lot of time, I'd think that we could skip the unittests for this PR

Reply:
This build was quicker than the previous build on master... What is really taking time is waiting for an OSX machine to be ready...